contract/scripts/run-chain.sh

```diff
@@ -1,68 +1,11 @@
 #!/bin/bash
 
-wait_for_bootstrap() {
-  endpoint="localhost"
-  while true; do
-    if json=$(curl -s --fail -m 15 "$endpoint:26657/status"); then
-      if [[ "$(echo "$json" | jq -r .jsonrpc)" == "2.0" ]]; then
-        if last_height=$(echo "$json" | jq -r .result.sync_info.latest_block_height); then
-          if [[ "$last_height" != "1" ]]; then
-            echo "$last_height"
-            return
-          else
-            echo "$last_height"
-          fi
-        fi
-      fi
-    fi
-    echo "waiting for next block..."
-    sleep 5
-  done
-  echo "done"
-}
-
-waitForBlock() (
```

Comment:
looks like `waitForBlock` is still in use. I see a call to it below.

Reply:
strange that ci passed.

Reply:
Noted. In `dapp-offer-up`, we are pulling the definition of `waitForBlock` from `/usr/src/upgrade-test-scripts/env_setup.sh` rather than defining it here. I would rather keep `run-chain.sh` identical in both dapps. Let me know which one is preferable.

Reply:
yes, let's keep it identical.